src/psyclone/transformations.py

```diff
@@ -2950,17 +2951,17 @@ class ACCDataTrans(RegionTrans):
 
     For example:
 
-    >>> from psyclone.parse import parse
+    >>> from psyclone.parse.algorithm import parse
```

Comment:
Similarly, would you mind trying doctest on this docstring too?

Reply:
Done... I've also added a kernels construct which I then surround with the data construct. This is strictly not needed and PSyclone will happily add the data construct around host code but... what's the point? Also, I've noticed the scalar 'r' isn't explicitly copied in, is that on purpose?

Reply:
Yes, we don't need to copy scalars. OpenACC does that automatically.